### PR TITLE
RFC: Refresh ARM64 guide

### PR DESCRIPTION
**Pull request by 72636c**
Follows on from #1350.

### CODE REVIEW

File: .changeset/fifty-toys-destroy.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Previously, the built-in templates made use of [`BUILDPLATFORM`](https://docs.do
 FROM --platform=${BUILDPLATFORM:-arm64} gcr.io/distroless/nodejs20-debian11
 ```
 
-1. Choose the platform of the host machine running the Docker build. A vCurrent SEEK build agent or Apple Silicon laptop will build under `arm64`, while an Intel laptop will build under `amd64`.
+1. Choose the platform of the host machine running the Docker build. A `vCurrent` Buildkite agent or Apple Silicon laptop will build under `arm64`, while an Intel laptop will build under `amd64`.
```
Comment (AaronMoat) [inline .changeset/fifty-toys-destroy.md:13]: Should we clarify this is just the default? x86 vCurrent is possible

**Review by AaronMoat — COMMENTED** [review]
Skimmed only so warrants another set of eyes but looks great

File: .changeset/fifty-toys-destroy.md (hunk 1 above)
Comment (72636c) [reply .changeset/fifty-toys-destroy.md:13]: Thanks for the catch, I initially missed that x86 vCurrent is the default and forgot to update this part.

**Review by etaoins — APPROVED** [review]
👏

**Pull request merged**